lib/configs/codeStyle.ts

```diff
@@ -1,17 +1,16 @@
+import type { ConfigOptions } from '../types.d.ts'
 /*!
  * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 import type { Linter } from 'eslint'
```

Comment:
Is there a way to avoid putting an SPDX header between imports?

Reply:
Good question. It makes sense to keep comments where there were as they could be for that import.
I will have a look if we can fix this.

Reply:
What about adding a rule that searches for the SPDX comment, checks for it's existence, and moves it to the top? And, maybe, checks the format.

Reply:
Actually, the solution is quite simple.
There should be an empty line between the SPDX header and the first import. Otherwise this comment is the import comment, not a file comment.

So, I'd call it a local code problem, not the config problem.